### ISSUE TITLE
The breadcrumb navigation is wrong.

### ISSUE THREAD

**Issue by slgxmh** [Lightning-AI/pytorch-lightning#20503]
### 📚 Documentation

<img width="795" alt="image" src="https://github.com/user-attachments/assets/de127acf-a716-48ad-af97-70c8018c9c77" />

As shown in above image.

**Comment by lantiga** [comment]
Good catch @slgxmh

**Comment by Borda** [comment]
yes that an issue and part of the story is the the parent page starts with accelerators
![image](https://github.com/user-attachments/assets/2f5727d8-32c2-44bb-953e-3d27077880d4)
and so when you do to subpage it preserve the fost title from parent page not the closest from a sections
